src/app/pages/svangerskapspenger/hvisDuSelvErSyk/HvisDuSelvErSyk.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { injectIntl, InjectedIntlProps } from 'react-intl';
+import PanelMedIllustrasjon from '../../../components/panel-med-illustrasjon/PanelMedIllustrasjon';
+import getTranslation from '../../../utils/i18nUtils';
+import Innhold, { getSource } from '../../../utils/innhold/Innhold';
+
+const seksjonsBilde = require('../../../assets/ark/ark-sykdom.svg').default;
+
+interface Props {
+    id: string;
+}
+
+const HvisDuSelvErSyk: React.StatelessComponent<Props & InjectedIntlProps> = ({ id, intl }) => {
+    return (
+        <PanelMedIllustrasjon
+            id={id}
+            title={getTranslation('om_svangerskapspenger.hvisDuSelvErSyk.tittel', intl)}
+            svg={seksjonsBilde}>
+            <Innhold
+                source={getSource('svangerskapspenger/hvis-du-selv-er-syk/hvisDuSelvErSyk', intl)}
```

Comment:
Skal vi holde på konvensjonen med lowercase og bindestreker i innholdsfilene? Altså `hvis-du-selv-er-syk` i stedet for `hvisDuSelvErSyk`

Reply:
Vi pleier vel også å skrive `æøå` i innholdsfiler og i nb_NO, nn_NO osv., skal vi prøve å holde det konsekvent? Sikkert ikke såå farlig da :)